hmpps-kotlin-spring-boot-test-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/test/kotlin/auth/HmppsAuthMockServer.kt

```diff
@@ -20,12 +20,10 @@ class HmppsAuthApiExtension : BeforeAllCallback, AfterAllCallback, BeforeEachCal
 
   override fun beforeAll(context: ExtensionContext) {
     hmppsAuth.start()
-    hmppsAuth.stubGrantToken()
```

Comment:
don't need to stub this unless calling out to other services so shouldn't be the default